Makefile

```
@@ -0,0 +1,40 @@
+MDBOOK_VERSION := v0.4.5
+SOURCE_PATH := docs/user-guide
+CONTAINER_RUNTIME := docker
```

In order to be able to override this, I had to change it to
```suggestion
CONTAINER_RUNTIME ?= docker
```